doc/releases/changelog-dev.md

```diff
@@ -15,22 +15,30 @@
   True
   ```
 
+<h4>New API for Qubit Mixed</h4>
+
+* Added `qml.devices.qubit_mixed` module for mixed-state qubit device support. This module introduces:
+
+  [(#6379)](https://github.com/PennyLaneAI/pennylane/pull/6379) An `apply_operation` helper function featuring:
+
+  * Two density matrix contraction methods using `einsum` and `tensordot`
+
+  * Optimized handling of special cases including: Diagonal operators, Identity operators, CX (controlled-X), Multi-controlled X gates, Grover operators
+
+  [(#6503)](https://github.com/PennyLaneAI/pennylane/pull/6503) A submodule 'initialize_state' featuring:
+
+  * A `density_matrix` function for initializing a density matrix from a state vector
+
+  * A `state_vector` function for initializing a state vector from a density matrix
+
+  * A `mixed_state` function for initializing a mixed state from a state vector
+
+  * A `state_vector_from_mixed` function for initializing a state vector from a mixed state
```

Comment:
Looks like in the `initialize_state.py` module there is only a `create_initial_state` function. Is this outdated?

Reply:
Good catch will fix